Stuck "Scuttling..." on 3.11.5 and 3.11.6

Since migrating to 3.11.5 my Patchwork seems to just sit there "Scuttling...".  I then installed 3.11.6 with the same results.

I cloned the repo and built it from source to find this in console:
`git-ssb is not installed, or not available in path`

afaik it should spawn a git-ssb server on its own, correct?  I had an older version installed a couple days ago and it worked fine.

I went ahead and did a manual `npm install git-ssb` and restarted the server to get this exception and stack trace:

```
Error: Cannot find module 'ssb-about'
    at Module._resolveFilename (module.js:543:15)
    at Function.Module._resolveFilename (D:\git\patchwork\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:31:8)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:74:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
```

There is also a deprecation notice when installing git-ssb:
```
npm WARN deprecated git-ssb@2.3.6: Installing git-ssb from centralized npm registry is deprecated; use ssb-npm
npm WARN deprecated git-remote-ssb@2.0.4: Installing git-remote-ssb from centralized npm registry is deprecated; use ssb-npm instead
npm WARN deprecated ssb-git-repo@2.8.3: Installing ssb-git-repo from centralized npm registry is deprecated; use ssb-npm instead
npm WARN @staltz/sbot-gossip@1.2.3 requires a peer of scuttlebot@>=11 but none is installed. You must install peer dependencies yourself.
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
```

I'm not familiar with the ssb-npm toolchain, but maybe this is the root of the issue with booting it on Windows.

> How long have you left it alone while it says "Scuttling"? Does it post any config output? I'm hoping for a bit more info to help figure out what's going on.

I left it for about 12 hours, seeing if there was some DB rebuild or something.  When that didn't work I wiped the .ssb directory and just restored my `gossip.json`, `manifest.json`, `secret`, and `ssb-secret.7z`.  I let that run for about 12 hours with no results.  Just "Scuttling..."  Where is the "config output" stored?  I poked around in .ssb looking for something human-readable but didn't come up with anything.

Here's what I get on an `npm start`:

```
{ path: 'C:\\Users\\Andy\\.ssb',
  party: true,
  timeout: 0,
  pub: true,
  local: true,
  friends: { dunbar: 150, hops: 2 },
  gossip: { connections: 3, autoPopulate: false },
  connections:
   { outgoing: { net: [Array] },
     incoming: { net: [Array], ws: [Array] } },
  timers: { connection: 0, reconnect: 5000, ping: 300000, handshake: 5000 },
  caps:
   { shs: '...',
     sign: null },
  master: [],
  logging: { level: 'notice' },
  port: 8008,
  blobsPort: 8989,
  server: true,
  _: [],
  host: '::',
  ws:
   { host: '::',
     port: 8989,
     scope: [ 'device', 'local', 'public' ],
     transform: 'shs' },
  keys:
   { curve: 'ed25519',
     public: '...',
     private: null,
     id: '...' },
  remote: 'net:127.0.0.1:undefined~...' }
Error: Cannot find module 'ssb-about'
    at Module._resolveFilename (module.js:543:15)
    at Function.Module._resolveFilename (D:\git\patchwork\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:31:8)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:74:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
```

From `npx ssb-server status`:
```
D:\git\patchwork\node_modules\ssb-server\bin.js
Error: Could not connect to ssb-server :::8008
Use the "start" command to start it.
```

Maybe a silly question, but have you tried restarting the app a few times? I've had it get stuck scuttling, but after a few minutes I'll restart and it'll work just fine. 

@funwhilelost 

This looks like your error:

```
Error: Cannot find module 'ssb-about'
    at Module._resolveFilename (module.js:543:15)
    at Function.Module._resolveFilename (D:\git\patchwork\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:473:25)
    at Module.require (module.js:586:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:31:8)
    at Object.<anonymous> (D:\git\patchwork\server-process.js:74:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
```

Can you verify that `~/.ssb/config` is empty? I'm curious whether maybe you modified something when you set up git-ssb. This is also concerning:

```
  remote: 'net:127.0.0.1:undefined~...' }
```

This looks like it doesn't know which port to connect on, so it's trying to connect on `undefined`? That looks broken to me, I'm curious whether you have the same output @lpar.

Seeing the same thing this morning after upgrading to `3.11.6` on macOS mojave. My error stack also started with `Error: Cannot find module 'ssb-about'`.

Downgraded to `3.11.4` and no errors in console, but it also seemingly stuck at "Scuttling".

--- a/index.js
+++ b/index.js
@@ -196,7 +196,7 @@ function setupContext (appName, opts, cb) {
 
   if (process.platform === 'win32') {
     // fix offline on windows by specifying 127.0.0.1 instead of localhost (default)
-    ssbConfig.remote = `net:127.0.0.1:${config.port}~shs:${pubkey}`
+    ssbConfig.remote = `net:127.0.0.1:${ssbConfig.port}~shs:${pubkey}`
   } else {
     const socketPath = Path.join(ssbConfig.path, 'socket')
     ssbConfig.connections.incoming.unix = [{ 'scope': 'device', 'transform': 'noauth' }]

